src/qt/walletview.cpp

```diff
@@ -197,6 +200,77 @@ void WalletView::gotoVerifyMessageTab(QString addr)
         signVerifyMessageDialog->setAddress_VM(addr);
 }
 
+void WalletView::gotoLoadPSBT()
+{
+    QString filename = GUIUtil::getOpenFileName(this,
+        tr("Load Transaction Data"), QString(),
+        tr("Partially Signed Transaction (*.psbt)"), nullptr);
+    if (filename.isEmpty()) return;
+    std::ifstream in(filename.toLocal8Bit().data(), std::ios::binary);
+    // https://stackoverflow.com/questions/116038/what-is-the-best-way-to-read-an-entire-file-into-a-stdstring-in-c
+    std::string data(std::istreambuf_iterator<char>{in}, {});
+
+    std::string error;
+    PartiallySignedTransaction psbtx;
+    if (!DecodeRawPSBT(psbtx, data, error)) {
```

Comment:
thought/note-to-myself: with the PR, we expose DecodeRawPSBT to possible broader userbase (not only RPC users but now also GUI users). Possible parsing exploits would then be a bigger risk.